fern/docs.yml

```diff
@@ -1,33 +1,90 @@
+# Main Fern configuration file
 instances:
   - url: privategpt.docs.buildwithfern.com
     custom-domain: docs.privategpt.dev
 
 title: PrivateGPT | Docs
 
+# The tabs definition, in the top left corner
+tabs:
+  overview:
+    display-name: Overview
+    icon: "fa-solid fa-home"
+  installation:
+    display-name: Installation
+    icon: "fa-solid fa-download"
+  manual:
+    display-name: Manual
+    icon: "fa-solid fa-book"
+  recipes:
+    display-name: Recipes
+    icon: "fa-solid fa-flask"
+  api-reference:
+    display-name: API Reference
+    icon: "fa-solid fa-file-contract"
+
+# Definition of tabs contents, will be displayed on the left side of the page, below all tabs
 navigation:
-  - section: Getting started
-    contents: 
-      - page: Welcome 
-        path: ./docs/pages/welcome.mdx
-      - page: Quickstart
-        path: ./docs/pages/quickstart.mdx
-      - page: Installation
-        path: ./docs/pages/installation.mdx
-      - page: LLMs
-        path: ./docs/pages/llms.mdx
-      - page: Vector Stores
-        path: ./docs/pages/vectordb.mdx
-  - section: Usage
-    contents:
-      - page: UI manual
-        path: ./docs/pages/ui.mdx
-      - page: Ingestion & Doc mgmt
-        path: ./docs/pages/ingestion.mdx
-      - page: SDKs
-        path: ./docs/pages/sdks.mdx
-  - api: API Reference
+  # The default tab
+  - tab: overview
+    layout:
+      - section: Welcome
+        contents:
+          - page: Welcome
+            path: ./docs/pages/overview/welcome.mdx
+          - page: Quickstart
+            path: ./docs/pages/overview/quickstart.mdx
+  # How to install privateGPT, with FAQ and troubleshooting
+  - tab: installation
+    layout:
+      - section: Getting started
+        contents:
+          - page: Installation
+            path: ./docs/pages/installation/installation.mdx
+  # Manual of privateGPT: how to use it and configure it
+  - tab: manual
+    layout:
+      - section: Ingestion
+        contents:
+          - page: Ingestion
+            path: ./docs/pages/manual/ingestion.mdx
+      - section: Document management
```

Comment:
I'd call this section Storage. 
In the future we'll talk here not only about Vector Stores but also about node storage options (disk vs database...)